### PR TITLE
Add instructions to embed code samples in docs

### PR DESCRIPTION
**Pull request by geriom**
# Changes

Add instructions to embed code samples from external files in the documentation.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by chitrangpatel — APPROVED** [review]
Looking forward to using this in https://github.com/tektoncd/pipeline/issues/6154

File: content/en/docs/Contribute/doc-con-formatting.md [hunk 1]
```diff
@@ -30,9 +30,57 @@ Tekton markdown formatting conventions:
 
 -   **Markdown files**
     - Hard-wrap the file lines so they are not longer than 80 characters.
-    - Use [reference-style links][ref-links] if the URL in the link is too
+    - Use [reference-style links][ref-links] if the URL if the link is too
```
Comment (AlanGreene) [inline content/en/docs/Contribute/doc-con-formatting.md:33]: 'in' was correct here, although I had to read it a couple of times. We could probably just have:
`Use [reference-style links][ref-links] if the URL is too long.`

i.e. drop 'in the link'

**Review by AlanGreene — CHANGES_REQUESTED** [review]
This looks good @geriom, thanks for documenting and providing examples. This will be very useful 😸 

One small change then I think it's good to go.

The red highlighting is a bit distracting but I don't think it's a blocker
image

File: content/en/docs/Contribute/doc-con-formatting.md (hunk 1 above)
Comment (geriom) [reply content/en/docs/Contribute/doc-con-formatting.md]: Oh, I think my brain ran into a race condition there 😅. Definitely no need to have both link and URL in that sentence.

Updated.

**Comment by geriom** [comment]
> The red highlighting is a bit distracting but I don't think it's a blocker

The code highlighter is doing that, it does look ugly. Switched language to `html` instead, looks better now.

**Review by AlanGreene — APPROVED** [review]

**Review by afrittoli — COMMENTED** [review]
/lgtm

**Pull request merged**